### ISSUE TITLE
Skupper-service-controller crash with 0.8.5

### ISSUE THREAD

**Issue by waldner** [skupperproject/skupper#671]
I get this error intermittently with 0.8.5:

```
2022/02/08 12:52:16 Skupper service controller
2022/02/08 12:52:16 Version: 0.8.5
2022/02/08 12:52:16 Setting up event handlers
2022/02/08 12:52:16 Waiting for Skupper router component to start
2022/02/08 12:52:21 Starting the Skupper controller
2022/02/08 12:52:21 Waiting for informer caches to sync
2022/02/08 12:52:21 Starting workers
E0208 12:52:21.211575       1 runtime.go:78] Observed a panic: runtime.boundsError{x:1, y:1, signed:true, code:0x0} (runtime error: index out of range [1] with length 1)
goroutine 107 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x160e000, 0xc000252a00)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:74 +0xa3
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:48 +0x82
panic(0x160e000, 0xc000252a00)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
main.addIngressBridge(0xc00011d950, 0xc00004a050, 0x24, 0xc000534620, 0x1, 0x0, 0x0)
	/go/src/app/cmd/service-controller/bridges.go:403 +0x84d
main.(*ServiceBindings).updateBridgeConfiguration(0xc00011d950, 0xc00004a050, 0x24, 0xc000534620)
	/go/src/app/cmd/service-controller/bridges.go:297 +0x72
main.requiredBridges(0xc00063e090, 0xc00004a050, 0x24, 0x16ce300)
	/go/src/app/cmd/service-controller/bridges.go:453 +0x129
main.(*Controller).updateBridgeConfig(0xc000286000, 0xc0002529a0, 0x19, 0x16e75d6, 0x1)
	/go/src/app/cmd/service-controller/controller.go:581 +0x171
main.(*Controller).processNextEvent.func1(0xc000286000, 0x14c82c0, 0xc000558c50, 0x0, 0x0)
	/go/src/app/cmd/service-controller/controller.go:732 +0x15a2
main.(*Controller).processNextEvent(0xc000286000, 0x0)
	/go/src/app/cmd/service-controller/controller.go:841 +0x79
main.(*Controller).runServiceCtrl(0xc000286000)
	/go/src/app/cmd/service-controller/controller.go:550 +0x2b
k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00061ac30)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:152 +0x5e
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00061ac30, 0x3b9aca00, 0x0, 0x1, 0xc000050a80)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:153 +0xf8
k8s.io/apimachinery/pkg/util/wait.Until(0xc00061ac30, 0x3b9aca00, 0xc000050a80)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:88 +0x4d
created by main.(*Controller).Run
	/go/src/app/cmd/service-controller/controller.go:309 +0x479
panic: runtime error: index out of range [1] with length 1 [recovered]
	panic: runtime error: index out of range [1] with length 1

goroutine 107 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:55 +2022-02-08T12:52:21.213931008Z 0x105
panic(0x160e000, 0xc000252a00)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
main.addIngressBridge(0xc00011d950, 0xc00004a050, 0x24, 0xc000534620, 0x1, 0x0, 0x0)
	/go/src/app/cmd/service-controller/bridges.go:403 +0x84d
main.(*ServiceBindings).updateBridgeConfiguration(0xc00011d950, 0xc00004a050, 0x24, 0xc000534620)
	/go/src/app/cmd/service-controller/bridges.go:297 +0x72
main.requiredBridges(0xc00063e090, 0xc00004a050, 0x242022-02-08T12:52:21.213939540Z , 0x16ce300)
	/go/src/app/cmd/service-controller/bridges.go:453 +0x129
main.(*Controller).updateBridgeConfig(2022-02-08T12:52:21.213946025Z 0xc000286000, 0xc0002529a0, 0x19, 0x16e75d6, 0x1)
	/go/src/app/cmd/service-controller/controller.go:581 +0x171
main.(*Controller).processNextEvent.func1(0xc000286000, 2022-02-08T12:52:21.213961050Z 0x14c82c0, 0xc000558c50, 0x0, 0x0)
	/go/src/app/cmd/service-controller/controller.go:732 +0x15a2
main.(*Controller).processNextEvent(0xc000286000, 0x0)
	2022-02-08T12:52:21.213975805Z /go/src/app/cmd/service-controller/controller.go:841 +0x79
main.(*Controller).runServiceCtrl(2022-02-08T12:52:21.213980840Z 0xc000286000)
	/go/src/app/cmd/service-controller/controller.go:550 +0x2b2022-02-08T12:52:21.213985831Z 
k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00061ac30)
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go2022-02-08T12:52:21.213991242Z :152 +0x5e
k8s.io/apimachinery/pkg/util/wait.JitterUntil(2022-02-08T12:52:21.213996766Z 0xc00061ac30, 0x3b9aca00, 0x0, 0x1, 0xc000050a802022-02-08T12:52:21.214001649Z )
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:153 +0xf8
k8s.io/apimachinery/pkg/util/wait.Until(0xc00061ac30, 0x3b9aca00, 0xc000050a802022-02-08T12:52:21.214011628Z )
	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:88 +0x4d
created by main.(*Controller).Run
	/go/src/app/cmd/service-controller/controller.go:2022-02-08T12:52:21.214021662Z 309 +0x479
```

"Intermittently" means that a fresh skupper deployment from manifest does not crash, but for example if I reboot the k8s cluster, when it comes back I get the above crash.

**Comment by galigian** [comment]
I've got the same issue.  Works fine until I restart the service controller and then fails the with the same index out of range error.

**Comment by HagaiBarel** [comment]
Same here, I'm running `0.8.4`

**Comment by grs** [comment]
Sorry folks! This should be fixed in 0.8.6

**Comment by ajssmith** [comment]
fixed in 0.8.6

**Comment by Kampe** [comment]
Hello!

We're seeing similar behavior still with 0.8.6:

```
2022-03-07T17:54:58.833369836Z stderr F 2022/03/07 17:54:58 Started workers
2022-03-07T17:54:58.633096277Z stderr F 2022/03/07 17:54:58 Console server listening on :8080
2022-03-07 11:54:58	
2022-03-07T17:54:58.432714333Z stderr F 2022/03/07 17:54:58 Starting workers
2022-03-07 11:54:58	
2022-03-07T17:54:58.332647803Z stderr F 2022/03/07 17:54:58 Waiting for informer caches to sync
2022-03-07 11:54:58	
2022-03-07T17:54:58.332592583Z stderr F 2022/03/07 17:54:58 Starting the Skupper controller
2022-03-07 11:54:58	
2022-03-07T17:54:53.321007763Z stderr F 2022/03/07 17:54:53 Waiting for Skupper router component to start
2022-03-07 11:54:58	
2022-03-07T17:54:53.314603333Z stderr F 2022/03/07 17:54:53 Setting up event handlers
2022-03-07 11:54:58	
2022-03-07T17:54:53.288709639Z stderr F 2022/03/07 17:54:53 Version: 0.8.6
2022-03-07 11:54:58	
2022-03-07T17:54:53.288654769Z stderr F 2022/03/07 17:54:53 Skupper service controller
2022-03-07 11:53:19	
2022-03-07T17:53:19.591033217Z stderr F 	/go/src/app/cmd/service-controller/controller.go:309 +0x479
2022-03-07 11:53:19	
2022-03-07T17:53:19.591028021Z stderr F created by main.(*Controller).Run
2022-03-07 11:53:19	
2022-03-07T17:53:19.59102298Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:88 +0x4d
2022-03-07 11:53:19	
2022-03-07T17:53:19.591018451Z stderr F k8s.io/apimachinery/pkg/util/wait.Until(0xc00044a080, 0x3b9aca00, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.591014077Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:153 +0xf8
2022-03-07 11:53:19	
2022-03-07T17:53:19.591007581Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00044a080, 0x3b9aca00, 0x0, 0x1, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590999788Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:152 +0x5e
2022-03-07 11:53:19	
2022-03-07T17:53:19.590986582Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00044a080)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590978151Z stderr F 	/go/src/app/cmd/service-controller/controller.go:550 +0x2b
2022-03-07 11:53:19	
2022-03-07T17:53:19.590957857Z stderr F main.(*Controller).runServiceCtrl(0xc00038a4b0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590952945Z stderr F 	/go/src/app/cmd/service-controller/controller.go:841 +0x79
2022-03-07 11:53:19	
2022-03-07T17:53:19.59094493Z stderr F main.(*Controller).processNextEvent(0xc00038a4b0, 0x1)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590940176Z stderr F 	/go/src/app/cmd/service-controller/controller.go:740 +0x1ba
2022-03-07 11:53:19	
2022-03-07T17:53:19.590934336Z stderr F main.(*Controller).processNextEvent.func1(0xc00038a4b0, 0x14c82c0, 0xc0002f8700, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590926772Z stderr F 	/go/src/app/cmd/service-controller/controller.go:575 +0x596
2022-03-07 11:53:19	
2022-03-07T17:53:19.590919763Z stderr F main.(*Controller).updateBridgeConfig(0xc00038a4b0, 0xc0006586c8, 0x1b, 0x1, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590913667Z stderr F 	/usr/local/go/src/runtime/panic.go:679 +0x1b2
2022-03-07 11:53:19	
2022-03-07T17:53:19.59090786Z stderr F panic(0x15354e0, 0x2400630)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590889074Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:55 +0x105
2022-03-07 11:53:19	
2022-03-07T17:53:19.590883564Z stderr F k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590877401Z stderr F goroutine 30 [running]:
2022-03-07 11:53:19	
2022-03-07T17:53:19.590870564Z stderr F 
2022-03-07 11:53:19	
2022-03-07T17:53:19.590864908Z stderr F [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x13bc876]
2022-03-07 11:53:19	
2022-03-07T17:53:19.590852891Z stderr F 	panic: runtime error: invalid memory address or nil pointer dereference
2022-03-07 11:53:19	
2022-03-07T17:53:19.590795741Z stderr F panic: runtime error: invalid memory address or nil pointer dereference [recovered]
2022-03-07 11:53:19	
2022-03-07T17:53:19.588176888Z stderr F 	/go/src/app/cmd/service-controller/controller.go:309 +0x479
2022-03-07 11:53:19	
2022-03-07T17:53:19.58816875Z stderr F created by main.(*Controller).Run
2022-03-07 11:53:19	
2022-03-07T17:53:19.588159228Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:88 +0x4d
2022-03-07 11:53:19	
2022-03-07T17:53:19.588150395Z stderr F k8s.io/apimachinery/pkg/util/wait.Until(0xc00044a080, 0x3b9aca00, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588143548Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:153 +0xf8
2022-03-07 11:53:19	
2022-03-07T17:53:19.588135895Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00044a080, 0x3b9aca00, 0x0, 0x1, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588128093Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:152 +0x5e
2022-03-07 11:53:19	
2022-03-07T17:53:19.588120118Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00044a080)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588113457Z stderr F 	/go/src/app/cmd/service-controller/controller.go:550 +0x2b
2022-03-07 11:53:19	
2022-03-07T17:53:19.588105498Z stderr F main.(*Controller).runServiceCtrl(0xc00038a4b0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588096734Z stderr F 	/go/src/app/cmd/service-controller/controller.go:841 +0x79
2022-03-07 11:53:19	
2022-03-07T17:53:19.588089714Z stderr F main.(*Controller).processNextEvent(0xc00038a4b0, 0x1)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588081538Z stderr F 	/go/src/app/cmd/service-controller/controller.go:740 +0x1ba
2022-03-07 11:53:19	
2022-03-07T17:53:19.58806568Z stderr F main.(*Controller).processNextEvent.func1(0xc00038a4b0, 0x14c82c0, 0xc0002f8700, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588058581Z stderr F 	/go/src/app/cmd/service-controller/controller.go:575 +0x596
2022-03-07 11:53:19	
2022-03-07T17:53:19.588050166Z stderr F main.(*Controller).updateBridgeConfig(0xc00038a4b0, 0xc0006586c8, 0x1b, 0x1, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588041002Z stderr F 	/usr/local/go/src/runtime/panic.go:679 +0x1b2
2022-03-07 11:53:19	
2022-03-07T17:53:19.588032801Z stderr F panic(0x15354e0, 0x2400630)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588024365Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:48 +0x82
2022-03-07 11:53:19	
2022-03-07T17:53:19.588016321Z stderr F k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588007213Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:74 +0xa3
2022-03-07 11:53:19	
2022-03-07T17:53:19.587997979Z stderr F k8s.io/apimachinery/pkg/util/runtime.logPanic(0x15354e0, 0x2400630)
2022-03-07 11:53:19	
2022-03-07T17:53:19.587983084Z stderr F goroutine 30 [running]:
2022-03-07 11:53:19	
2022-03-07T17:53:19.587918001Z stderr F E0307 17:53:19.587709       1 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)		2022-03-07 11:54:59	
2022-03-07T17:54:58.833369836Z stderr F 2022/03/07 17:54:58 Started workers
2022-03-07 11:54:58	
2022-03-07T17:54:58.633096277Z stderr F 2022/03/07 17:54:58 Console server listening on :8080
2022-03-07 11:54:58	
2022-03-07T17:54:58.432714333Z stderr F 2022/03/07 17:54:58 Starting workers
2022-03-07 11:54:58	
2022-03-07T17:54:58.332647803Z stderr F 2022/03/07 17:54:58 Waiting for informer caches to sync
2022-03-07 11:54:58	
2022-03-07T17:54:58.332592583Z stderr F 2022/03/07 17:54:58 Starting the Skupper controller
2022-03-07 11:54:58	
2022-03-07T17:54:53.321007763Z stderr F 2022/03/07 17:54:53 Waiting for Skupper router component to start
2022-03-07 11:54:58	
2022-03-07T17:54:53.314603333Z stderr F 2022/03/07 17:54:53 Setting up event handlers
2022-03-07 11:54:58	
2022-03-07T17:54:53.288709639Z stderr F 2022/03/07 17:54:53 Version: 0.8.6
2022-03-07 11:54:58	
2022-03-07T17:54:53.288654769Z stderr F 2022/03/07 17:54:53 Skupper service controller
2022-03-07 11:53:19	
2022-03-07T17:53:19.591033217Z stderr F 	/go/src/app/cmd/service-controller/controller.go:309 +0x479
2022-03-07 11:53:19	
2022-03-07T17:53:19.591028021Z stderr F created by main.(*Controller).Run
2022-03-07 11:53:19	
2022-03-07T17:53:19.59102298Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:88 +0x4d
2022-03-07 11:53:19	
2022-03-07T17:53:19.591018451Z stderr F k8s.io/apimachinery/pkg/util/wait.Until(0xc00044a080, 0x3b9aca00, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.591014077Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:153 +0xf8
2022-03-07 11:53:19	
2022-03-07T17:53:19.591007581Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00044a080, 0x3b9aca00, 0x0, 0x1, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590999788Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:152 +0x5e
2022-03-07 11:53:19	
2022-03-07T17:53:19.590986582Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00044a080)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590978151Z stderr F 	/go/src/app/cmd/service-controller/controller.go:550 +0x2b
2022-03-07 11:53:19	
2022-03-07T17:53:19.590957857Z stderr F main.(*Controller).runServiceCtrl(0xc00038a4b0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590952945Z stderr F 	/go/src/app/cmd/service-controller/controller.go:841 +0x79
2022-03-07 11:53:19	
2022-03-07T17:53:19.59094493Z stderr F main.(*Controller).processNextEvent(0xc00038a4b0, 0x1)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590940176Z stderr F 	/go/src/app/cmd/service-controller/controller.go:740 +0x1ba
2022-03-07 11:53:19	
2022-03-07T17:53:19.590934336Z stderr F main.(*Controller).processNextEvent.func1(0xc00038a4b0, 0x14c82c0, 0xc0002f8700, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590926772Z stderr F 	/go/src/app/cmd/service-controller/controller.go:575 +0x596
2022-03-07 11:53:19	
2022-03-07T17:53:19.590919763Z stderr F main.(*Controller).updateBridgeConfig(0xc00038a4b0, 0xc0006586c8, 0x1b, 0x1, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590913667Z stderr F 	/usr/local/go/src/runtime/panic.go:679 +0x1b2
2022-03-07 11:53:19	
2022-03-07T17:53:19.59090786Z stderr F panic(0x15354e0, 0x2400630)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590889074Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:55 +0x105
2022-03-07 11:53:19	
2022-03-07T17:53:19.590883564Z stderr F k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.590877401Z stderr F goroutine 30 [running]:
2022-03-07 11:53:19	
2022-03-07T17:53:19.590870564Z stderr F 
2022-03-07 11:53:19	
2022-03-07T17:53:19.590864908Z stderr F [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x13bc876]
2022-03-07 11:53:19	
2022-03-07T17:53:19.590852891Z stderr F 	panic: runtime error: invalid memory address or nil pointer dereference
2022-03-07 11:53:19	
2022-03-07T17:53:19.590795741Z stderr F panic: runtime error: invalid memory address or nil pointer dereference [recovered]
2022-03-07 11:53:19	
2022-03-07T17:53:19.588176888Z stderr F 	/go/src/app/cmd/service-controller/controller.go:309 +0x479
2022-03-07 11:53:19	
2022-03-07T17:53:19.58816875Z stderr F created by main.(*Controller).Run
2022-03-07 11:53:19	
2022-03-07T17:53:19.588159228Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:88 +0x4d
2022-03-07 11:53:19	
2022-03-07T17:53:19.588150395Z stderr F k8s.io/apimachinery/pkg/util/wait.Until(0xc00044a080, 0x3b9aca00, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588143548Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:153 +0xf8
2022-03-07 11:53:19	
2022-03-07T17:53:19.588135895Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc00044a080, 0x3b9aca00, 0x0, 0x1, 0xc0000ac660)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588128093Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/wait/wait.go:152 +0x5e
2022-03-07 11:53:19	
2022-03-07T17:53:19.588120118Z stderr F k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1(0xc00044a080)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588113457Z stderr F 	/go/src/app/cmd/service-controller/controller.go:550 +0x2b
2022-03-07 11:53:19	
2022-03-07T17:53:19.588105498Z stderr F main.(*Controller).runServiceCtrl(0xc00038a4b0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588096734Z stderr F 	/go/src/app/cmd/service-controller/controller.go:841 +0x79
2022-03-07 11:53:19	
2022-03-07T17:53:19.588089714Z stderr F main.(*Controller).processNextEvent(0xc00038a4b0, 0x1)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588081538Z stderr F 	/go/src/app/cmd/service-controller/controller.go:740 +0x1ba
2022-03-07 11:53:19	
2022-03-07T17:53:19.58806568Z stderr F main.(*Controller).processNextEvent.func1(0xc00038a4b0, 0x14c82c0, 0xc0002f8700, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588058581Z stderr F 	/go/src/app/cmd/service-controller/controller.go:575 +0x596
2022-03-07 11:53:19	
2022-03-07T17:53:19.588050166Z stderr F main.(*Controller).updateBridgeConfig(0xc00038a4b0, 0xc0006586c8, 0x1b, 0x1, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588041002Z stderr F 	/usr/local/go/src/runtime/panic.go:679 +0x1b2
2022-03-07 11:53:19	
2022-03-07T17:53:19.588032801Z stderr F panic(0x15354e0, 0x2400630)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588024365Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:48 +0x82
2022-03-07 11:53:19	
2022-03-07T17:53:19.588016321Z stderr F k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
2022-03-07 11:53:19	
2022-03-07T17:53:19.588007213Z stderr F 	/go/pkg/mod/k8s.io/apimachinery@v0.17.0/pkg/util/runtime/runtime.go:74 +0xa3
2022-03-07 11:53:19	
2022-03-07T17:53:19.587997979Z stderr F k8s.io/apimachinery/pkg/util/runtime.logPanic(0x15354e0, 0x2400630)
2022-03-07 11:53:19	
2022-03-07T17:53:19.587983084Z stderr F goroutine 30 [running]:
2022-03-07 11:53:19	
2022-03-07T17:53:19.587918001Z stderr F E0307 17:53:19.587709       1 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
```

Apologize for the log formatting, Loki currently has no way to output to a file.
